pubspec.yaml

```diff
@@ -19,7 +19,7 @@ dev_dependencies:
   build_web_compilers: ^3.0.0
   dart_dev: ^3.8.5
   dart_style: ^2.1.1
-  dependency_validator: '>=2.0.1 <4.0.0'
+  dependency_validator: ^3.2.2
```

Comment:
This package is far enough upstream that we can update to 3.2.2, which is the version with the NNBD fix.